lightly/loss/swav.py

```diff
@@ -0,0 +1,133 @@
+from typing import List
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.distributed as dist
+
+
+@torch.no_grad()
+def sinkhorn(out: torch.Tensor, iterations: int = 3, epsilon: float = 0.05):
+    """Distributed sinkhorn algorithm.
+
+    Args:
+        out:
+            Similarity of the features and the SwaV prototypes.
+        iterations:
+            Number of sinkhorn iterations.
+        epsilon:
+            Temperature parameter.
+
+    Returns:
+        Soft codes Q assigning each feature to a prototype.
+    
+    """
+
+    # get the world size in the distributed case
```

Comment:
I would for now drop the distributed part

Reply:
why?